<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QWidget" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Тип отображения ребер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_lines_type">
         <item>
          <property name="text">
           <string>Сплошная</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string> Пунктир</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Толщина ребер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_width_lines"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Тип отображения вершин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_vertices_type">
         <item>
          <property name="text">
           <string>Не отображать</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Круглые вершины</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Квадратные вершины</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Размер вершин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_vertices_size"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Тип отображения проекции</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_central">
         <property name="text">
          <string>Центральная</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_parallel">
         <property name="text">
          <string>Параллельная</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>55</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Цвет ребер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Цвет ребер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Цвет вершин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_color_vertices">
         <property name="text">
          <string>Цвет вершин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Цвет полигонов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_color_polygon">
         <property name="text">
          <string>Цвет полигонов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Цвет фона</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_color_background">
         <property name="text">
          <string>Цвет фона </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="button_accept">
           <property name="text">
            <string>Принять</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_cancel">
           <property name="text">
            <string> Сбросить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
